### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by anne-decusatis** [anne-decusatis/genderamender#26]
Now that #7 is done, we can add some new languages. 
As gender is so complicated, I feel it's important to get gender options in each language written by people who are familiar with that culture. Using Google Translate or similar has a high chance of being offensive and/or inaccurate. 

It would also be great if we could have some examples and tests in non-English, once we get the word translations sorted out.

**Comment by coraliepac** [comment]
I contributed for French (I uploaded a JSON file but can get any format since I used a TMS).

In 3 words: not that simple :) And I think you've very right to not rely on machine translation for this one because this is as complicated as translation can get, and some machine translated terms are highly inaccurate or offensive (example: Google offers _asexué_ for _genderless_ which is in my opinion not inclusive and carries a pejorative meaning and generally relates to the fact of having sex with people or not). 

I think to summary the problem with translating these, I will quote [this article](http://srishtimadurai.blogspot.com/2013/07/pains-of-minority-among-minorities-gopi.html) from Gopi Shankar, a gender activist in India who was the first to propose actual real translations/transliterations for gender studies terms, in his native language, Tamil: 

> Even within the mainstream LGBT community in India, the existence of these many genders is largely unknown.

Most of these concepts (and gender studies in general) are largely unknown/ignored in the world and consequently don't have existing, accepted translations (They will come as this culture gets imported/globalization/some would say _americanization_ etc.). You're touching the very bottom of the problem with translation here. Words and the way things are named emanate from local cultures. 

From a translation point of view, what happens in French (France) for example:
1/ A lot would stay in English, until they become a bit more mainstream locally and get an accepted, visible translation
2/ Some already have a visible / accepted translation, in which case it can be used
3/ Some are actually gender terms stolen from French language to be used in English with a different meaning (I'll use Italic when I refer to the French word here). The best example is Femme. _Femme_ is the French word that means Woman/Female in French. Looks like it was stolen to be used for something else in English. It can get confusing. It seems like there is an accepted terminology among the French community to use the word _Fem_ to describe the idea, concept of Femme. There is a bit of research to do here among the community.
4/ For some I'm hesitant, like Female vs. Woman. Technically there is only one French word here, _Femme._ We could say _Femelle_ for _Female_ but that's kind of pejorative, since _Femelle_ is used strictly for animals in French from France. Most likely, Female and Woman = _Femme_.
5/ Androgyne/androgynous also only has one possible translation (_androgyne_) since _Androgyne_ means Androgynous in French, English stealing from French again... one solution would be to keep _Androgynous_ in English if we want to differentiate it from Androgyne.
6/ There may be things that exist in the local language and that are not on your list.
I think something interesting and innovative would be to include some non-US gender concepts here. Thailand and Thai culture has[ interesting local gender-related ideas](https://en.wikipedia.org/wiki/Gender_identities_in_Thailand). Look at [Hijra ](https://en.wikipedia.org/wiki/Hijra_%28South_Asia%29) in India, [Fa'afafine](https://en.wikipedia.org/wiki/Fa%27afafine) in Polynesia... The terminology ['Third sex' ](https://en.wikipedia.org/wiki/Third_gender) is a commonly, rather positive and accepted terminology in France... (maybe thanks to the immense popularity of Simone de Beauvoir's _Deuxième sexe_?) and generally seems to translate well... 

some food for thought  :)

I guess in some languages like Japanese that would be mostly untranslatable and everything would stay in English (or, maybe transliterated as Katakana characters) and be considered American concepts. I think an interested language to work with would be Brazilian Portuguese, which I find good at importing, welcoming with terminology originating from the US (you could ask Victor). 

**Resources:**
- Facebook gender selection tool in French, they seem to have been putting some thoughts into the translations for these
- Wikipedia [Genderqueer page in French](https://fr.wikipedia.org/wiki/Genderqueer) contains the most information about the terms used locally
- [Slate](http://www.slate.fr/culture/83605/52-genre-facebook-definition)
- [Femme Wikipedia page in French](https://fr.wikipedia.org/wiki/Butch-fem)
- Generally looked around Wikipedia translation of US English pages
- Somebody wrote a full research article on the [difficulty of translating gender studies related terms](https://ilcea.revues.org/832) in French.

tl;dr: translation is hard.

**Comment by anne-decusatis** [comment]
wow, thank you so much @bonjourcora! I didn't expect everything to have a one-to-one mapping (since a lot of the English words mean similar things) but I also didn't realize these concepts would be referred to with English words in other languages. This is a really interesting and thought-provoking response, I really appreciate you taking the time to write it!

as a side note, I don't see your JSON file :( happy to do what I can to help - you can copy the file here: https://github.com/anne-decusatis/genderamender/new/master/genders and then click the button at the bottom that says something like "create a new branch and new pull request". Then I can look at the pull request. Since I know you in-person, I can sit with you while you do it if you want, if any part of that is confusing, just let me know!

**Comment by coraliepac** [comment]
I think it's here now: https://github.com/anne-decusatis/genderamender/pull/27
Sorry, as a non-engineer I'm not always familiar to Github/pull request etc. As a reward for my efforts you can give a mini-course? : - )
I was really happy to help! thanks

**Comment by patch** [comment]
I've found that the Wikipedia interlanguage links can be a great resource for translations because they've been manually linked by humans.

For example, the [Genderqueer](https://en.wikipedia.org/wiki/Genderqueer) article has interlanguage links for 19 languages, including the local title of each article in the `title` attribute of the `a` element. To find them, search for `interlanguage-link` in the source. This would still need to be reviewed, but could be used to generate a helpful base set of potential translations, which could significantly reduce the amount of manual investigation for any language.
- Genderqueer – Basque (`eu`)
- Genderqueer – Danish (`da`)
- Genderqueer – Finnish (`fi`)
- Genderqueer – French (`fr`)
- Genderqueer – Italian (`it`)
- Genderqueer – Portuguese (`pt`)
- Genderqueer – Spanish (`es`)
- Genderqueer – Tagalog (`tl`)
- Intergender – Swedish (`sv`)
- Interrodnost – Serbo-Croatian (`sh`)
- Neduuma genro – Esperanto (`eo`)
- Rhyngrywedd – Welsh (`cy`)
- Гендерквир – Russian (`ru`)
- Гендерквір – Ukrainian (`uk`)
- ג'נדרקוויר – Hebrew (`he`)
- جنسیت کوییر – Persian (`fa`)
- 젠더퀴어 – Korean (`ko`)
- ジェンダークィア – Japanese (`ja`)
- 性別酷兒 – Chinese (`zh`)

I created a script to scrape and compile interlanguage link titles from Wikipedia, as well as to crawl through all the pages listed in a category or list page. I'm happy to help apply the script to this project. It would be fun to get together and hack on this if you're interested!

**Comment by anne-decusatis** [comment]
@patch that's really cool, thanks for sharing! I would love your contribution, and I would love to get together and hack sometime! (The next time I have on my calendar dedicated to hacking is the next MergeSort event, but if for whatever reason you can't make it then, send me an email, we'll find a time.)
